Documentation/QuickInstall/Composer/Index.rst

```diff
@@ -44,6 +44,11 @@ Point the document root of your web server to the `public` folder.
 If you do not want to use Composer, follow the :ref:`package installation
 guide <get-and-unpack-the-typo3-package>`.
 
+See also
+========
+
+* `Tutorial - Quick TYPO3 Installation with Composer <https://www.youtube.com/watch?v=M4imQOUKDe4>`_ from the official TYPO3 YouTube channel
```

Comment:
@pixeldesu You can directly embed YouTube videos in reST. Has advantage that people are not directed away from docs. 

See https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/YoutubeVideos.html

You can find the id in the URL.

Reply:
@pixeldesu Slightly related (about Composer): I added some text for composer.json to "TYPO3 Explained". Oddly enough, that had been missing. If you could review the PR, that would be very helpful, see https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/pull/556

Reply:
I will adjust the video to show instead of being a link.

Another question before I start the change, should I also rename the heading because it now becomes somewhat specific to just the video?

The "See also" is more of a list of links to places with information, so if the full video is displayed there it doesn't really fit the purpose anymore, should I rename it to something like "Tutorial from the official TYPO3 YouTube channel"?

Reply:
Added the requested changes just now!

Reply:
About your question: I think you found a good solution already!